### PR TITLE
Unit tests setup

### PR DESCRIPTION
**Pull request by PiotrZakrzewski**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -37,6 +37,8 @@ before_script:
 script:
   - grails refresh-dependencies --non-interactive
   - grails test-app --non-interactive --stacktrace
+  - chmod a+x test/runit/run_tests.R
```
Comment (cataphract) [inline .travis.yml]: You don't need this line. Only if you wanted to run like this `./test/runit/run_tests.R`

**Comment by cataphract** [comment]
You have 10 commits for 30 lines, with corrections in between. Rebase this in one commit.

File: .travis.yml (hunk 1 above)
Comment (PiotrZakrzewski) [inline .travis.yml]: It fails without it

Comment (cataphract) [inline .travis.yml]: Just tested, ran fine without it. I'm merging the amended commit.

**Pull request merged**